diff --git a/src/nvcore/Debug.cpp b/src/nvcore/Debug.cpp
index 9ab4525..ffe5d2d 100644
--- a/src/nvcore/Debug.cpp
+++ b/src/nvcore/Debug.cpp
@@ -1020,8 +1020,12 @@ void debug::dumpCallstack(MessageHandler *messageHandler, int callstackLevelsToS
         Array<const char *> lines;
         writeStackTrace(trace, size, callstackLevelsToSkip + 1, lines);     // + 1 to skip the call to dumpCallstack
 
+        va_list empty;
+        va_start(empty, callstackLevelsToSkip);
+        va_end(empty);
+
         for (uint i = 0; i < lines.count(); i++) {
-            messageHandler->log(lines[i], NULL);
+            messageHandler->log(lines[i], empty);
             delete lines[i];
         }
     }
